docs/install.rst

```diff
@@ -3,40 +3,42 @@
 Installation
 ============
 
+``pyirf`` requires Python ≥3.7 and pip, plus the packages defined in
+the ``setup.py``.
 
-``pyirf`` requires Python ≥3.6 and the packages as defined in the ``setup.py``.
 Core dependencies are
 
 * ``numpy``
 * ``astropy``
 * ``scipy``
 
-Dependencies for development, like unit tests and building the documentation
-are defined in ``extras``.
+We provide an environment file for Anaconda or Miniconda users.
 
 Installing a released version
 -----------------------------
 
-
-To install a release version, just install the ``pyirf`` package using
+To install a released version, just install the ``pyirf`` package using
 
 .. code-block:: bash
 
     pip install pyirf
 
 or add it to the dependencies of your project.
 
-
 Installing for development
 --------------------------
 
-If you want to work on pyirf itself, clone the repository or your fork of
-the repository and install the local copy of pyirf in development mode.
+If you want to work on pyirf itself, clone the repository and install the local
+copy of pyirf in development mode.
+
+It is required that the python environment contains also ``cython`` before
```

Comment:
Please add that this is only required temporarily because of an issue with gammapy and add numpy, which is also required at install time.

Reply:
right I forgot